textx/metamodel.py

```diff
@@ -164,6 +165,7 @@ def __init__(self, file_name=None, classes=None, builtins=None,
 
         super(TextXMetaModel, self).__init__(**kwargs)
 
+        self._tx_model_param_definitions = ModelParamDefinitions()
```

Comment:
This looks like a good place to introduce textX built-in model param definitions.

Reply:
ok. also adapated tests